src/lib/components/Identifiers/IdentifiersField.js

```diff
@@ -62,13 +62,11 @@ export class IdentifiersField extends Component {
                   />
                 )}
                 <Form.Field>
-                  <Form.Field>
-                    <label>&nbsp;</label>
-                    <Button
-                      icon="close"
-                      onClick={() => arrayHelpers.remove(indexPath)}
-                    />
-                  </Form.Field>
+                  <label>&nbsp;</label>
```

Comment:
not a blocker: I would solve it by a padding instead of adding the empty label, it does not look super clean